src/Backend/Session.vala

```diff
@@ -472,7 +472,7 @@ public class Mail.Backend.Session : Camel.Session {
         return null;
     }
 
-    public async void save_draft (Camel.MimeMessage message, Camel.InternetAddress from, Camel.Address recipients) throws Error {
+    public async void save_draft (Camel.MimeMessage message, Camel.InternetAddress from, Camel.Address recipients, string? ancestor_message_info_uid = null) throws Error {
```

Comment:
```suggestion
    public async void save_draft (Camel.MimeMessage message, Camel.InternetAddress from, Camel.Address recipients, Camel.MessageInfo? ancestor_message_info = null) throws Error {
```
Any reason we can't directly use the right messageinfo object here? (this would avoid a lookup)

Reply:
@tintou good catch! Passing `Camel.MessageInfo` directly works as expected.